perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1008,6 +1009,8 @@ sub process_user : Private {
             $user->name($name) unless $user->name;
             $c->stash->{no_reporter_alert} = 1;
         } elsif ($c->stash->{contributing_as_another_user}) {
+            # TODO Is this block reachable?
```

Comment:
Yes, in the circumstance given by the comment below - if neither username nor phone are provided.